### PR TITLE
Add json file for post

### PR DESCRIPTION
**Pull request by rm2013**

### CODE REVIEW

**Comment by mattjonesorg** [comment]
This would be very helpful for our requests that can be large at times.

**Comment by missedone** [comment]
looks quite similar with PR #68

File: boom.go [hunk 1]
```diff
@@ -85,6 +87,7 @@ Options:
   -T  Content-type, defaults to "text/html".
   -a  Basic authentication, username:password.
   -x  HTTP Proxy address as host:port.
+  -f  file to read for post
```
Comment (rakyll) [inline boom.go]: -f File to use as the request body.

**Comment by rakyll** [comment]
The project has moved to https://github.com/rakyll/hey. Closing the PRs on this legacy repo.

**Pull request closed without merging**